services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -27,9 +30,14 @@ class WriteMetadataActor(override val batchSize: Int,
   private val statsRecorder = MetadataStatisticsRecorder(metadataStatisticsRecorderSettings)
 
   override def process(e: NonEmptyVector[MetadataWriteAction]) = instrumentedProcess {
+    val config = ConfigFactory.load().getConfig("services.MetadataService.config")
+    val metadataKeysToCleanOption = config.as[Option[List[String]]]("metadata-keys-to-sanitize-utf8mb4")
+    val metadataKeysToClean = metadataKeysToCleanOption.getOrElse(List())
```

Comment:
I see that some of the config settings are being passed in to `MetadataWriteActor` at creation time - can we do the same with this one? See `MetadataServiceActor` around line 108.